fastseq/models/__init__.py

```diff
@@ -3,6 +3,7 @@
 """Register models"""
 
 import logging
+import sys
```

Comment:
What's this for?

Reply:
This is required for exception at line 19 
https://github.com/microsoft/fastseq/blob/a21d906171869c813b28a72297ac6e00a3712979/fastseq/models/__init__.py#L19

Reply:
Gotcha. Thanks!